packages/plugin-core/assets/dendron-ws/tutorial/treatments/quickstart-skip-welcome/tutorial.md

```diff
@@ -1,69 +1,73 @@
 ---
 id: c1bs7wsjfbhb0zipaywqins
-title: Quickstart
+title: Tutorial
 desc: ""
-updated: 1658180875356
+updated: 1658342369402
 created: 1654223767390
 currentStep: 0
 totalSteps: 0
 ---
 
-Welcome to Dendron! Dendron is a developer-focused knowledge base that helps you manage information using flexible hierarchies!
+Welcome to Dendron! Dendron is a developer-focused knowledge base that helps you manage information using **flexible hierarchies**!
 
 You are currently in the tutorial vault (a vault is the folder where your notes are stored). Feel free to edit this note and create new files as you go through the quickstart!
 
 ## Create a Note
 
 1. Use `Ctrl+L` / `Cmd+L` to bring up the lookup prompt
-1. Type 'dendron' and press `<ENTER>`
-1. Congrats, you just created your first note!
+1. Type `dendron` and select `Create New`
+
+- > NOTE: After you press enter, Dendron will create and open the `dendron` note. Use `<ALT>-<TAB>/<WINDOW>-<TAB>` to come back to this note
```

Comment:
What keybinding is this supposed to be on mac?  ctrl-tab goes back to the last tab in vscode on a mac

Reply:
thanks for the catch. it is `ctrl-tab` not `alt-tab`. made corrections